File: backend/src/main/java/xyz/engsmyre/moviedescriptiongame/repository/GameSessionRepository.java
package xyz.engsmyre.moviedescriptiongame.repository;

import java.util.Optional;
import java.util.UUID;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import xyz.engsmyre.moviedescriptiongame.db.GameSessionMovies;

@Repository
public interface GameSessionRepository extends CrudRepository<GameSessionMovies, UUID> {
    Optional<GameSessionMovies> findGameSessionMoviesBySessionId(UUID gameId);
}

File: backend/src/main/java/xyz/engsmyre/moviedescriptiongame/repository/GameSessionRepository.kt
package xyz.engsmyre.moviedescriptiongame.repository

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import xyz.engsmyre.moviedescriptiongame.db.GameSessionMovies
import java.util.*

@Repository
interface GameSessionRepository : CrudRepository<GameSessionMovies, UUID?> {
    fun findGameSessionMoviesBySessionId(gameId: UUID?): Optional<GameSessionMovies>
}